packages/terra-site/src/examples/search-field/SearchFieldBasic.jsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 import SearchField from 'terra-search-field';
 
-const view = () => (
-  <div>
-    <SearchField />
-  </div>
-);
+class SearchFieldView extends React.Component {
 
-export default view;
+  constructor(props) {
+    super(props);
+    this.state = { searchText: '' };
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Search Text: {this.state.searchText}</p>
+        <SearchField onSearch={(searchText) => { this.setState({ searchText }); }} />
```

Comment:
The debounce from the SearchField will disable the callback for updating the state when the character count falls bellow the minimum. I don't know if this is correct or incorrect depending on what the paragraph above this line is trying to display